### PR TITLE
Adding a new interpreter file

### PR DESCRIPTION
**Pull request by RobbieMcKinstry**
Works for simple hello world applications. I'm unsure about more complex code.
https://github.com/wyvernlang/wyvern/issues/44

### CODE REVIEW

File: tools/mac_script.bash [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+CLASSPATH=$HOME/workspace/wyvern/tools/lib/ant.jar:$HOME/workspace/wyvern/tools/lib/junit-4.11.jar:$HOME/workspace/wyvern/tools/lib/asm-debug-all-5.0.1.jar:$HOME/workspace/wyvern/tools/lib/CopperCompiler.jar:$HOME/workspace/wyvern/tools/lib/CopperRuntime.jar:$HOME/workspace/wyvern/tools/lib/hamcrest-core-1.3.jar:$HOME/workspace/wyvern/tools/lib/javatuples-1.2.jar:$HOME/workspace/wyvern/tools/build:.
```
Comment (JonathanAldrich) [inline tools/mac_script.bash]: Some details of this are specific to your setup, e.g. that Wyvern is in $HOME/workspace.  So why don't you either make it generic or leave it out of the PR?

File: tools/src/wyvern/tools/Interpreter.java [hunk 2]
```diff
@@ -7,47 +8,45 @@
...
+            ExpressionAST ast = (ExpressionAST) TestUtil.getNewAST(source);
+            Expression program = ast.generateIL(GenContext.empty());
+            TypeContext ctx = TypeContext.empty();
+            program.typeCheck(ctx);
+            Value v = program.interpret(EvalContext.empty());
+            System.out.println(v.toString());
+        } catch(ParseException e) {
+            System.err.println("Failed to parse file.");
+            e.printStackTrace();
+        }
```
Comment (JonathanAldrich) [inline tools/src/wyvern/tools/Interpreter.java]: This is fine, but even better would be to also catch any ToolError (e.g. a typechecking error) and print the error message on standard error.

File: tools/mac_script.bash (hunk 1 above)
Comment (RobbieMcKinstry) [inline tools/mac_script.bash]: Whoops! This was accidental! It shouldn't be in here, sorry!

**Comment by JonathanAldrich** [comment]
Thanks, looks good!

**Pull request merged**